wallet/udb/txmined.go

```diff
@@ -3384,6 +3384,30 @@ func (s *Store) balanceFullScan(ns, addrmgrNs walletdb.ReadBucket, minConf int32
 			}
 			votingAuthorityAmt := dcrutil.Amount(0)
 			lockedByTicketsAmt := dcrutil.Amount(0)
+
+			// Calculate total input amount which will allow a proper fee calculation.
+			// Fee is needed to be removed from the stake.AmountFromSStxPkScrCommitment
+			// due to the way tickets are constructed and rewards are calculated.
+			totalInputAmount := dcrutil.Amount(0)
+			for i := range rec.MsgTx.TxIn {
+				_, credKey, err := existsDebit(ns,
+					&txHash, uint32(i), &blockRec.Block)
+				if err != nil {
+					return err
+				}
+				if credKey == nil {
+					continue
+				}
+				credVal := existsRawCredit(ns, credKey)
+				if credVal == nil {
+					return apperrors.New(apperrors.ErrNoExist, "couldn't find a credit for unspent txo")
```

Comment:
Should use ErrData here.  ErrNoExists actually has a completely different meaning.